pkg/containerd/cmd/service/system_init.go

```diff
@@ -71,6 +76,32 @@ func systemInit(args []string) {
 		os.Exit(1)
 	}
 
+	// remove (unlikely) old containerd socket
+	_ = os.Remove(*sock)
+
+	// start up containerd
+	cmd := exec.Command(*binary)
+	cmd.Stderr = os.Stderr
```

Comment:
well I hope it never does output on stdout, as that would be most incorrect, but I guess it does no harm changing it...